### PR TITLE
Verification image

### PR DESCRIPTION
**Pull request by LinnierGames**
- Use the mapview to allow the user to tap on it and view the directions [trello](https://trello.com/c/UHHwjpd7)
> The feature to use the user's location has been moved [here](https://trello.com/c/zlkzBkDP)

### CODE REVIEW

File: KifuSF/KFCDelivery.swift [hunk 1]
```diff
@@ -151,14 +151,40 @@ class KFCDelivery: KFCModularTableView {
...
     
+    override func didSelect(_ cellType: KFCModularTableView.CellTypes, at indexPath: IndexPath) {
+        switch cellType {
+        case .destinationMap:
+            guard let delivery = self.delivery else {
+                return assertionFailure("No delivery given while map was tappable")
+            }
+            
+            if delivery.status.isAwaitingPickup {
+
```
Comment (pondorasti) [inline KifuSF/KFCDelivery.swift:173]: nit: extra whitespace

File: KifuSF/KFCDelivery.swift [hunk 2]
```diff
@@ -151,14 +151,40 @@ class KFCDelivery: KFCModularTableView {
...
+                return assertionFailure("No delivery given while map was tappable")
+            }
+            
+            if delivery.status.isAwaitingPickup {
+                
+                //pickup Location
+                MapHelper(long: delivery.longitude, lat: delivery.latitude)
+                    .open()
+            } else if delivery.status.isAwaitingDelivery {
+
```
Comment (pondorasti) [inline KifuSF/KFCDelivery.swift:178]: nit: extra whitespace

**Comment by pondorasti** [comment]
Your PR, looks good, but I have a few things that I think should be fixed while we are here: add missing marks.

File: KifuSF/KFCDelivery.swift (hunk 1 above)
Comment (LinnierGames) [reply KifuSF/KFCDelivery.swift:173]: This is a style for me: I leave a leading line for every comment i write so I can find them better.

**Comment by LinnierGames** [comment]
[Submit Verification Image](https://trello.com/c/UHHwjpd7/112-submit-verification-image)

**Review by pondorasti — APPROVED** [review]
The PR looks good

**Pull request merged**